components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/browser/BrowserModel.java

```diff
@@ -44,16 +45,22 @@
 import org.openmicroscopy.shoola.agents.imviewer.view.ImViewer;
 import org.openmicroscopy.shoola.agents.imviewer.view.ImViewerFactory;
 import org.openmicroscopy.shoola.agents.imviewer.view.ViewerPreferences;
+
 import ome.model.units.BigResult;
 import omero.log.LogMessage;
+
 import org.openmicroscopy.shoola.env.LookupNames;
+
 import omero.model.Length;
```

Comment:
while fiddling with these, may want to keep the org ones together and the omero ones together?